addon/utils/bugsnag-configuration.js

```diff
@@ -14,21 +14,27 @@ export default class BugsnagConfiguration {
         bugsnagInstance[key] = this.config[key];
       });
     } else {
+      /* eslint-disable no-console */
```

Comment:
The default in ember is now eslint, and the default config fails on `console`s. We could either allow `console` everywhere or do what I did here which is to excuse specific instances. Without this, tests fail.